contrib/release.sh

```diff
@@ -6,7 +6,7 @@
 set -euox pipefail
 
 main () {
-    for crate in "internals" "hashes" "bitcoin"; do
+    for crate in "bitcoin" "hashes" "internals" "units"; do
```

Comment:
SSOT violation. Why don't we just have a script that pulls the metadata from cargo and then uses that?

Reply:
We could use the `contrib/crates.sh` script we have already.

Reply:
Should we hold up merging over this?

Reply:
Lets just merge as is. When we do the 1.0 releases this script is totally wrong even if we fix the SSOT problem since the 1.0 crates will have different requirements to the pre-1.0

Reply:
Sure, this is not a blocking issue but we have these bugs over and over. For 1.0 crates we should just check the version of each crate (when released) and change behavior based on that.